<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleMessenger</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxClientServer">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButtonClientMode">
           <property name="text">
            <string>Run in Client mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelClient">
           <property name="text">
            <string>Program will connect to an other instance running in Server mode</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayoutClient">
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelClientHost">
             <property name="text">
              <string>Host (IP address) :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditClientHost">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="placeholderText">
              <string>ex: 90.0.188.37</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelClientPort">
             <property name="text">
              <string>Port :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBoxClientPort">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>7777</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonClientConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButtonServerMode">
           <property name="text">
            <string>Run in Server mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelServer">
           <property name="text">
            <string>Program will wait for a an other instance running in Client mode to connect</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayoutServer">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelServerPort">
             <property name="text">
              <string>Listen on port :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBoxServerPort">
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>7777</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Set port to 0 to choose automatically</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonServerStart">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutMessageSend">
      <item>
       <widget class="QLineEdit" name="lineEditMessageSend"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMessageSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutSimpleMessenger"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAboutSimpleMessenger">
   <property name="text">
    <string>About SimpleMessenger</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
